pants.toml

```diff
@@ -0,0 +1,28 @@
+[GLOBAL]
+pants_version = "2.10.0.dev1"
+backend_packages = [
+  "pants.backend.codegen.protobuf.python",
+  "pants.backend.experimental.codegen.thrift.apache.python",
+  "pants.backend.python",
+]
+
+[source]
+root_patterns = ["src/*"]
+
+[anonymous-telemetry]
+enabled = true
+repo_id = "205E301D-44A4-42FF-8495-C9D41871A431"
+
+[python]
+interpreter_constraints = ["CPython>=3.8,<3.10"]
+
+[python-protobuf]
+runtime_dependencies = [
+  "//3rdparty/python:protobuf",
+]
+
+[thrift-python]
```

Comment:
I should probably rename this in Pants to `python-thrift` to match `python-protobuf`. Thoughts on what convention to use?

Reply:
I'm in favour of `python-X` for all things python. That makes them group nicely in docs etc when sorting on title and similar.

Reply:
Probably `python-thrift`, as people tend to think of Python as the primary concept, thrift as the secondary one within their python code.

Reply:
+1